README.md

```diff
@@ -3,15 +3,16 @@ To Start Servers:
 Proxy : npm run server-dev
 freddie-individual-item-details : npm run server
 kimarie-bonuses : npm run start
+max-descriptions : npm run server-dev
```

Comment:
I would use `npm run start` for my service.

Reply:
Danke